### PR TITLE
Update iOS docs with regards to how to use APNS entitlements

### PR DESCRIPTION
**Pull request by dmipon**
Update docs wrt to Automatic entitlements on 4.x and 5.x
Update reference only for 5.x

### CODE REVIEW

File: docs/voice/voice-for-ios/voice-ios-push-notifications-callkit.md [hunk 1]
```diff
@@ -210,7 +210,7 @@ This means how the the app is code signed and what _Provisioning Profile_ is use
 
 For example, if your application is signed with a _Development_ provisioning profile it will be bound to the APS _Development_ environment. If it’s code signed with a _Distribution_ provisioning profile it will be bound to the APS _Production_ environment.
 
-Typically a _Debug_ build will be code signed with a _Development_ provisioning profile and thus `SINAPSEnvironmentDevelopment` should be used. And typically a _Release_ build will be code signed with a _Distribution_ provisioning profile and thus `SINAPSEnvironmentProduction` should be used. Instead of changing this manually for each build, the macro `SINAPSEnvironmentAutomatic` is available which automatically expands to _Development_ for _Debug_ builds and _Production_ for _Release_ builds.
+Typically a _Debug_ build will be code signed with a _Development_ provisioning profile and thus `SINAPSEnvironmentDevelopment` should be used. And typically a _Release_ build will be code signed with a _Distribution_ provisioning profile and thus `SINAPSEnvironmentProduction` should be used. **You are responsible for selecing proper entitlements depending on your build type and signing profile.**
```
Comment (GiovanniBeriSinch) [inline docs/voice/voice-for-ios/voice-ios-push-notifications-callkit.md:213]: typo: "selecing"->"selecting", in all occurrences of this sentence

File: docs/tutorials/tutorials-ios/ios-managed-push-tutorial.md [hunk 2]
```diff
@@ -80,9 +80,9 @@ We’re currently managing the Sinch client from our AppDelegate, so we’re goi
...
 ```objectivec
-self.push = [Sinch managedPushWithAPSEnvironment:SINAPSEnvironmentAutomatic];
-  self.push.delegate = self;
-  [self.push setDesiredPushTypeAutomatically];
+self.push = [Sinch managedPushWithAPSEnvironment:SINAPSEnvironmentDevelopment];
+self.push.delegate = self;
+[self.push setDesiredPushTypeAutomatically];
 ```
 
 Here you can see we’re setting the APS Environment to automatic, our delegate, and then the desired push type to automatic. When automatic is available, use it\!
```
Comment (GiovanniBeriSinch) [inline docs/tutorials/tutorials-ios/ios-managed-push-tutorial.md:88]: "we’re setting the APS Environment to automatic"

You should update this part as well

File: docs/video/video-for-ios/video-ios-local-and-remote-push-notifications.md [hunk 3]
```diff
@@ -274,4 +274,4 @@ Certificates configured with Sinch can be replaced or renewed by uploading new o
 When an iOS application is code signed, it is the embedded _Provisioning Profile_ that will specify which _Apple Push Notification Service Environment_ (_APS Environment_) the acquired push notification device token will be bound to. Depending on how an application is provisioned it has an effect on what should be passed to `[Sinch managedPushWithAPSEnvironment:]`. For example if your application is signed with a _Development_ provisioning profile it will be bound to the APS _Development_ environment. If it’s code signed with a _Distribution_ provisioning profile (also referred to as _Universal_) it will be bound
 to the APS _Production_ environment.
```
Comment (GiovanniBeriSinch) [inline docs/video/video-for-ios/video-ios-local-and-remote-push-notifications.md:275]: not related to your change: this newline shouldn't be here

File: docs/tutorials/tutorials-ios/ios-managed-push-tutorial.md (hunk 2 above)
Comment (GiovanniBeriSinch) [reply docs/tutorials/tutorials-ios/ios-managed-push-tutorial.md:88]: and what about the "automatic push type"? Is it still valid?

**Review by GiovanniBeriSinch — CHANGES_REQUESTED** [review]

File: docs/tutorials/tutorials-ios/ios-managed-push-tutorial.md (hunk 2 above)
Comment (dmipon) [reply docs/tutorials/tutorials-ios/ios-managed-push-tutorial.md:88]: `automatic push type` is still part of 4.2.5 so i will leave it

**Comment by GiovanniBeriSinch** [comment]
If if checkout your branch and I `grep` for `SINAPSEnvironmentAutomatic` I see there are a bunch of occurrences in `docs/*/reference/html/...` folder:
```
└─ rg -i SINAPSEnvironmentAutomatic
docs/video/video-for-ios/reference/html/Constants/SINAPSEnvironment.html
95:<p>The macro <code>SINAPSEnvironmentAutomatic</code> can be used to specify SINAPSEnvironment based on the type of build.

docs/video/video-for-ios/reference/html/Protocols/SINManagedPush.html
120:    self.push = [Sinch managedPushWithAPSEnvironment:SINAPSEnvironmentAutomatic]

docs/voice/voice-ios-cloud/reference/html/Constants/SINAPSEnvironment.html
95:<p>The macro <code>SINAPSEnvironmentAutomatic</code> can be used to specify SINAPSEnvironment based on the type of build.

docs/voice/voice-for-ios/reference/html/Protocols/SINManagedPush.html
120:    self.push = [Sinch managedPushWithAPSEnvironment:SINAPSEnvironmentAutomatic]

docs/voice/voice-for-ios/reference/html/Constants/SINAPSEnvironment.html
95:<p>The macro <code>SINAPSEnvironmentAutomatic</code> can be used to specify SINAPSEnvironment based on the type of build.
```
Do you know if you also have to take care of that?

**Comment by dmipon** [comment]
@GiovanniBeriSinch It is refense documentation that is generated from Source code. We will take care of that when we update SDK to newer version

**Review by GiovanniBeriSinch — APPROVED** [review]

**Comment by reddvl1980** [comment]
> @GiovanniBeriSinch It is reference documentation that is generated from Source code. We will take care of that when we update SDK to newer version

@dmipon , if changes are needed in the reference, let's update it here as well.

**Comment by GiovanniBeriSinch** [comment]
All changes of last commit are generated automatically, right. If this is the case, LGTM

**Pull request merged**